packages/embark/src/lib/modules/parity/check.js

```diff
@@ -0,0 +1,47 @@
+const WebSocket = require("ws");
```

Comment:
Can we move this to `embark-utils`? The reason the original file lives where it does is so that we could iterate faster, but this check is the same for all Ethereum providers that adhere to the protocol.

Reply:
I'd like to see some stuff abstracted out as well (a la https://github.com/embark-framework/embark/pull/1741), but there is a view point that too much abstraction can be bad. We could (in my opinion, should) have a broader discussion on this topic.